src/backend/efiling-api/src/test/jag-efiling-api.postman_collection.json

```diff
@@ -206,7 +206,7 @@
             ],
             "body": {
               "mode": "raw",
-              "raw": "{\n    \"navigationUrls\": {\n        \"success\": \"http//somewhere.com\",\n        \"error\": \"http//somewhere.com\",\n        \"cancel\": \"http//somewhere.com\"\n    },\n    \"clientAppName\": \"my app\",\n    \"filingPackage\": {\n        \"court\": {\n        \"location\": \"1211\",\n        \"level\": \"P\",\n        \"courtClass\": \"F\"\n    },\n    \"documents\": [\n        {\n            \"name\": \"test.pdf\",\n            \"type\": \"WNC\",\n            \"description\": \"Without Notice Application Checklist\",\n            \"statutoryFeeAmount\": 0\n        }\n    ]\n    }\n}"
+              "raw": "{\n    \"navigationUrls\": {\n        \"success\": \"http//somewhere.com\",\n        \"error\": \"http//somewhere.com\",\n        \"cancel\": \"http//somewhere.com\"\n    },\n    \"clientAppName\": \"my app\",\n    \"filingPackage\": {\n        \"court\": {\n            \"location\": \"1211\",\n            \"level\": \"P\",\n            \"courtClass\": \"F\"\n        },\n        \"documents\": [\n            {\n                \"name\": \"test.pdf\",\n                \"type\": \"WNC\",\n                \"description\": \"Without Notice Application Checklist\",\n                \"statutoryFeeAmount\": 0,\n                \"data\": {},\n                \"md5\": \"string\"\n            }\n        ],\n        \"parties\": [\n            {\n                \"partyType\": \"IND\",\n                \"roleType\": \"CLA\",\n                \"firstName\": \"first\",\n                \"middleName\": \"middle\",\n                \"lastName\": \"last\"\n            }\n        ]\n    }\n}"
```

Comment:
Description does not exist I think, it's coming from old version of the api probably 🤣